web/src/main/java/org/akaza/openclinica/controller/ImportController.java

```diff
@@ -130,10 +121,10 @@ public ImportController() {
 
             odmContainer = (ODMContainer) um1.unmarshal(reader);
 
-        } catch (Exception me1) {
-            me1.printStackTrace();
-            logger.info("found exception with xml transform");
-            return new ResponseEntity(ErrorConstants.ERR_INVALID_XML_FILE + "\n" + me1.getMessage(), HttpStatus.UNSUPPORTED_MEDIA_TYPE);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Can we remove this print statement and add logger.error("..." + e) instead of just printing the message. That will log the entire stacktrace.

Reply:
All set , thanks